cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORGenerator.java

```diff
@@ -138,6 +150,8 @@ public int getMask() {
 
     protected boolean _cfgMinimalInts;
 
+    protected boolean _cfgLenientUnicodeEncoding;
```

Comment:
Brief javadoc would be nice (I know, some other entries do not have it yet but... )

Reply:
Done